ethicalads-theme/templates/ea/includes/advertisers-inbound.html

```diff
@@ -82,6 +82,11 @@ <h2 class="font-weight-bold">Advertise across our network</h2>
                     </div>
                   </div>
 
+                  <div class="form-group">
+                    <label for="advertising-goals">What are your advertising goals?</label>
+                    <textarea class="form-control" id="advertising-goals" name="advertising-goals" placeholder="I'm trying to promote the free trial of my widget that helps with Docker and Kubernetes deployments to DevOps developers."></textarea>
```

Comment:
```suggestion
                    <textarea class="form-control" id="advertising-goals" name="advertising-goals" placeholder="I'm trying to promote the free trial of my product that helps with Docker and Kubernetes deployments to DevOps developers."></textarea>
```

widget feels a bit dismissive, at least to me.